README.md

```diff
@@ -123,7 +123,7 @@ node .
 
 The best option when in development, is to compile and run all at once.  It will then keep watching the source files to check for changes and, if there is one, it will re-compile and relaunch the app.
 
-This way, you don't need to compile it first, and it will keep the app running updated versions when you change the source files. 
+This way, you don't need to compile it first, and it will keep the app running updated versions when you change the source files.
 
```

Comment:
It took me a while to find the changed line at the end, which is not good, I almost thought you have the option to trim useless whitespace at the end of several lines and that was the only change. If you burden your reviewers with too many changes, it gets tedious. 

There are two ways to fix this in a team.  If all use the same editor or IDE with the same settings (in this case, the option of whether to trim or not the spurious whitespace, these pointless changed files would not happen.  The other is when you review the files in your *commit*.  If you see in the *commit list* a file you didn't change, or many changes that you didn't mean to have,  you might not include it in the commit and/or  reverse the unintended changes. 

If you have no control over what each contributor uses for an editor, you add a *Contributor guidelines* file in the repository.  Another alternative is have a [`editorconfig`](https://editorconfig.org/) file in the root directory.  Many editors and IDEs understand this file and use it as a *workspace settings* for the editor.  VSCode is one.  Finally, for JS  code, there is [Prettier](https://prettier.io/).    And, at any rate, a commit with files with too many spurious changes will not be accepted, it is bad manners to submit such a change.

Reply:
I see, I was also wondering why it marked this as a change even though I didn't touch that line... I use Atom to open the file so what you are explaining is probably the cause of this change.